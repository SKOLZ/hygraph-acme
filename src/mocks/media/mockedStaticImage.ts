export const mockedStaticImage = {
  __typename: "StaticImage" as const,
  image: {
    url: "https://media.graphassets.com/bbd3PKNcTBGVkNbcois7",
    width: 480,
    height: 320,
  },
};
