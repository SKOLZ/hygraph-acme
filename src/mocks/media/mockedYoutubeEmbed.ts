export const mockedYoutubeEmbed = {
  __typename: "YoutubeEmbed" as const,
  title: "Media Title",
  youtubeUrl: "https://www.youtube.com/embed/znkZ8X_7tLA",
};
