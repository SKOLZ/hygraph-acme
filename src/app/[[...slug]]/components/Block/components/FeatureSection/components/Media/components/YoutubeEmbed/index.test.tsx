import { render } from "@testing-library/react"
import { mockedYoutubeEmbed } from "@/mocks/media/mockedYoutubeEmbed"
import { YoutubeEmbed } from "."

describe("YoutubeEmbed", () => {
  it("renders the element correctly", () => {
    const {container} = render(<YoutubeEmbed {...mockedYoutubeEmbed} />);
    expect(container).toMatchSnapshot();
  })
})
