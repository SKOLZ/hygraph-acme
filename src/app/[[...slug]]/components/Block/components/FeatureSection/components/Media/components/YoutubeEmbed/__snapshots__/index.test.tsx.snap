// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`YoutubeEmbed renders the element correctly 1`] = `
<div>
  <iframe
    allowfullscreen=""
    class="aspect-video lg:m-10"
    loading="lazy"
    src="https://www.youtube.com/embed/znkZ8X_7tLA"
    title="Media Title"
    width="100%"
  />
</div>
`;
